docs/components/InputFile/Web.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import {
+  FileUpload,
+  InputFile,
+  updateFiles,
+} from "@jobber/components/InputFile";
+import { FormatFile } from "@jobber/components/FormatFile";
+import { Heading } from "@jobber/components/Heading";
+
+export default {
+  title: "Components/Forms and Inputs/InputFile/Web",
+  component: InputFile,
+  parameters: {
+    viewMode: "story",
+    // CAN THIS BE SET FOR A PARTICULAR STORY?
+    // options: { showPanel: false },
+    previewTabs: {
+      code: {
+        hidden: false,
+        extraImports: {
+          "@std-proposal/temporal": ["CivilTime"],
```

Comment:
You will want to update: https://github.com/GetJobber/atlantis/blob/master/.storybook/components/Playground/constants.ts to include the dependency and version

Reply:
Lol... This is just here because I copied and modified `InputGroup` for my template... but I think we want to still make the update you recommend for `InputGroup`

Reply:
I'll make it in this PR.